pages/getting_started/collect.rst

```diff
@@ -0,0 +1,73 @@
+================
+Collect Messages
+================
+
+Once Graylog and associated components are running, the next step is to begin collecting logs.
+
+The first step is to create an input. Inputs define the method by which Graylog collects logs. Out of the box, Graylog supports multiple methods to collect logs, including:
+
+* Syslog (TCP, UDP, AMQP, Kafka)
+* GELF(TCP, UDP, AMQP, Kafka, HTTP)
+* AWS - AWS Logs, FlowLogs, CloudTrail
+* Beats/Logstash
+* CEF (TCP, UDP, AMQP, Kafka)
+* JSON Path from HTTP API
+* Netflow (UDP)
+* Plain/Raw Text (TCP, UDP, AMQP, Kafka)
+
+Content packs
+=============
+
+Additional inputs may be installed via content packs. Content packs are bundles of Graylog input, extractor, stream, dashboard, and output configurations that can provide full support
+for a data source. Some content packs are shipped with Graylog by default and some are available from the website. Content packs that
+were downloaded from `the Graylog Marketplace <http://marketplace.graylog.org>`__ can be imported using the Graylog web interface.
+
+You can load and even create your own content packs from the ``System -> Content Packs`` section of your Graylog web interface.
```

Comment:
we have in the complete docs ``System / Content Packs`` and not ``System -> Content Packs`` 

Just to have it the same